app/page/template/modals.htm

```diff
@@ -264,7 +264,7 @@
       </div>
       <div class="modal-text">
         <span data-bind="l10n_text: z.string.modal_session_reset_message_1"></span>
-        <a data-bind="l10n_href: z.string.url_support_contact_bug, l10n_text: z.string.modal_session_reset_message_link" target="_blank"></a>
+        <a rel="nofollow noopener noreferrer" data-bind="l10n_href: z.string.url_support_contact_bug, l10n_text: z.string.modal_session_reset_message_link" target="_blank"></a>
```

Comment:
as it's less important and closely tied with the `target` maybe putting it just before `target`?